packages/block-library/src/comments/edit/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+import { useBlockProps, useInnerBlocksProps } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import CommentsInspectorControls from './comments-inspector-controls';
+import CommentsLegacy from './comments-legacy';
+import TEMPLATE from './template';
+
+export default function CommentsEdit( props ) {
+	const { attributes, setAttributes } = props;
+	const { tagName: TagName, legacy } = attributes;
+
+	const blockProps = useBlockProps( {
+		// We add the previous block name for backward compatibility.
+		className: 'wp-block-comments-query-loop',
```

Comment:
I realize [this was around](https://github.com/WordPress/gutenberg/blob/b4d23a7e5bc4aa28917415bed70d05462e63d7dc/packages/block-library/src/comments/edit.js#L95-L96) before this PR, but I'm not totally sure it should be there -- might be an oversight in #40506. (We should probably file a separate PR to remove this.)

Reply:
(We should first merge https://github.com/WordPress/gutenberg/pull/42335 and then rebase this PR to carry over the change.)

Reply:
#42335 merged; I'll rebase this PR ASAP.